src/runtime/set.cpp

```diff
@@ -600,6 +652,87 @@ Box* setGt(BoxedSet* self, BoxedSet* rhs) {
     return setIssuperset(self, rhs);
 }
 
+Box* setContains(BoxedSet* self, Box* v) {
+    RELEASE_ASSERT(PyAnySet_Check(self), "");
+
+    if (PySet_Check(v)) {
```

Comment:
Hmm I'm not sure I understand the logic of this block.  It also seems to give a different result from CPython for:
```python
print set() in frozenset([frozenset()])
```